sinch/domains/voice/__init__.py

```diff
@@ -0,0 +1,180 @@
+)
+from sinch.domains.voice.models.calls.requests import GetVoiceCallRequest, UpdateVoiceCallRequest
+from sinch.domains.voice.models.calls.responses import GetVoiceCallResponse
+
+
+class Callouts:
+    def __init__(self, sinch):
+        self._sinch = sinch
+
+    def text_to_speech(
+        self,
+        destination: dict,
+        cli: str = None,
+        dtmf: str = None,
+        domain: str = None,
+        custom: str = None,
+        locale: str = None,
+        text: str = None,
+        prompts: str = None,
+        enable_ace: bool = None,
+        enable_dice: bool = None,
+        enable_pie: bool = None
+    ) -> VoiceCalloutResponse:
+        return self._sinch.configuration.transport.request(
+            CalloutEndpoint(
+                callout_method=CalloutMethod.TTS.value,
+                request_data=TextToSpeechVoiceCalloutRequest(
+                    destination=destination,
+                    cli=cli,
+                    dtmf=dtmf,
+                    domain=domain,
+                    custom=custom,
+                    locale=locale,
+                    text=text,
+                    prompts=prompts,
+                    enableAce=enable_ace,
+                    enableDice=enable_dice,
+                    enablePie=enable_pie
+                )
+            )
+        )
+
+    def conference(
+        self,
+        destination: dict,
+        conference_id: str,
+        cli: str = None,
+        conference_dtmf_options: dict = None,
+        dtmf: str = None,
+        conference: str = None,
+        max_duration: int = None,
+        enable_ace: bool = None,
+        enable_dice: bool = None,
+        enable_pie: bool = None,
+        locale: str = None,
+        greeting: str = None,
+        moh_class: str = None,
+        custom: str = None,
+        domain: str = None
+    ) -> VoiceCalloutResponse:
+        return self._sinch.configuration.transport.request(
+            CalloutEndpoint(
+                callout_method=CalloutMethod.CONFERENCE.value,
+                request_data=ConferenceVoiceCalloutRequest(
+                    destination=destination,
+                    conference_id=conference_id,
+                    cli=cli,
+                    conferenceDtmfOptions=conference_dtmf_options,
+                    dtmf=dtmf,
+                    conference=conference,
+                    maxDuration=max_duration,
+                    enableAce=enable_ace,
+                    enableDice=enable_dice,
+                    enablePie=enable_pie,
+                    locale=locale,
+                    greeting=greeting,
+                    mohClass=moh_class,
+                    custom=custom,
+                    domain=domain
+                )
+            )
+        )
+
+    def custom(
+        self,
+        cli: str = None,
+        destination: dict = None,
+        dtmf: str = None,
+        custom: str = None,
+        max_duration: int = None,
+        ice: str = None,
```

Comment:
It would be super nice for DX to provide helpers for SVAML building: asking to user to write inline escaped JSON is not the best experience ☹️

Reply:
I'm tackling SVAML in the next sprint. I need to figure out a pythonic compromise for this feature.